packages/smooth_app/lib/pages/product/summary_card.dart

```diff
@@ -567,4 +621,43 @@ class _SummaryCardState extends State<SummaryCard> {
       ),
     );
   }
+
+  bool allowAttributeOpening(Attribute attribute) =>
+      widget.isFullVersion &&
+      widget.knowledgePanels != null &&
+      attribute.panelId != null;
+
+  Future<void> openFullKnowledgePanel({
+    required final Attribute attribute,
+    required bool done,
+    required KnowledgePanels? knowledgePanels,
+  }) async {
+    if (!allowAttributeOpening(attribute)) {
+      return;
+    }
+
+    if (!done) {
+      knowledgePanels = await LoadingDialog.run<KnowledgePanels>(
```

Comment:
What does this loading dialog look like ? Does it just say "Loading". That might be ok but I'm just checking if we want a custom message ?